### PR TITLE
Add deps support

### PR DESCRIPTION
**Pull request by clyfe**
Add deps.edn, same rationale as https://github.com/weavejester/integrant/issues/61

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{:paths   ["src"]
+ :deps    {org.clojure/clojure         {:mvn/version "1.10.1"}
+           org.clojure/tools.namespace {:mvn/version "1.0.0"}
+           integrant/integrant         {:mvn/version "1.0.0"}}
```
Comment (weavejester) [inline deps.edn]: The Integrant version is wrong.

File: deps.edn [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+ :aliases {:test {:extra-deps  {org.clojure/clojurescript {:mvn/version "1.10.773"}}
```
Comment (weavejester) [inline deps.edn]: As this is a PR against `master`, the ClojureScript part should be removed for now.

**Review by weavejester — CHANGES_REQUESTED** [review]
Thanks for the PR. Some corrections are required, and the commit message should probably be something more like "Add deps.edn" as "deps support" is ambiguous.

File: deps.edn (hunk 2 above)
Comment (clyfe) [reply deps.edn]: Removed aliases.

File: deps.edn (hunk 1 above)
Comment (clyfe) [reply deps.edn]: Changed to "0.8.0".

File: deps.edn (hunk 2 above)
Comment (weavejester) [reply deps.edn]: Can you fix the spacing too?

Comment (clyfe) [reply deps.edn]: Fixed spacing.

**Pull request merged**